ttnn/cpp/ttnn/decorators.hpp

```diff
@@ -297,7 +298,9 @@ struct registered_operation_t {
     template <typename... args_t>
     auto operator()(args_t&&... args) const {
         tt::log_debug(tt::LogOp, "Started   C++ ttnn operation: {}", std::string_view{cpp_fully_qualified_name});
+        GraphTracker::instance().track_begin_op(cpp_fully_qualified_name, args...);
         auto output = invoke(std::forward<args_t>(args)...);
+        GraphTracker::instance().track_end_op(output);
```

Comment:
any concerns if there is an exception?

Reply:
from other side if it throws we never get output.

Reply:
We need to review it in a future, because there are more places like this and I think it will not be worse.